Examples/LocationManager/Common/AppCore.swift

```diff
@@ -225,4 +225,11 @@ extension PointOfInterest {
       && lhs.subtitle == rhs.subtitle
       && lhs.title == rhs.title
   }
+
+  public func hash(into hasher: inout Hasher) {
```

Comment:
Could you move this down to the bottom of this file? That's where we also have the `==` implementation, and it just helps remove any unnecessary noise from the top of this file so people can focus on the important parts.

Reply:
This is already at the correct location I believe 👍

Reply:
Oh sorry! I misread the diff. Thanks!